data/lang/jp.json

```diff
@@ -1397,54 +1397,54 @@
-  "config_boosuccess" : "Successfully assigned/removed allowed channels for boosters!",
-  "stanalyzer_sid" : "Please specify ID of the map you want to analyze",
-  "stanalyzer_code": "Please specify code of the map you want to analyzer (ex : S, RA, NA, etc.)",
-  "stanalyzer_range" : "The range you gave exceeds total number of stage in this map",
-  "stanalyzer_analyze" : "Analyzing stage... : %d / %d",
-  "permission_file" : "Attaching Files",
-  "missing_permission" : "Bot noticed that it lacks specific permissions which are required to perform this command\nPlease contact moderator to adjust permission, or if you are one of them, please add permissions below to bot\n\n----------\n\n_PPP_\n\n---------\n\nThis message is from server : **_SSS_**\nChannel : **_CCC_**",
-  "permission_managemsg" : "Managing Messages (Deletion, etc.)",
-  "permission_addreact" : "Adding Reactions",
-  "permission_addrole" : "Managing Roles (Adding, Deleting, etc.)",
-  "permission_addemoji" : "Managing Emojis and Stickers (Adding, Deleting, etc.)",
-  "permission_externalemoji" : "Using External Emojis",
-  "comban_noid" : "You have to decide which member to ban from using this bot!",
-  "comban_nonumber" : "You have to provide either mention or ID of user",
-  "comban_nomember" : "Bot couldn't find such user in this server",
-  "comban_already" : "This user is already banned from using bot in this server",
-  "comban_confirm" : "Are you sure that you want to ban this user <@_> from using PackPack? Once you ban this user, they won't be able to use any commands of PackPack",
-  "comban_success" : "Successfully banned user <@_>",
-  "comban_fail" : "Failed to get member data...",
-  "comban_noself" : "You can't ban yourself!",
-  "comunban_notban" : "This user isn't banned from using this bot",
-  "comunban_confirm" : "Are you sure that you want to unban this user <@_> from using PackPack? This means user will be able to use commands of PackPack back",
-  "comunban_success" : "Successfully unbanned user <@_>",
-  "ignorechp_nomem" : "This server doesn't have assigned member role yet",
+  "config_forfalse" : "通常、個人の`config`は`serverconfig`より優先されます。しかし、チャットが長くなるのでモデレーターはコンパクトモードの使用を強制させることができます\n\nBotは`serverconfig`を無視して個人の`config`を使用します",
+  "config_trueform" : "第三形態検索モード : %s",
+  "config_truetrue" : "第三形態検索モードではBotが第三形態を自動で選択します(第三形態がない場合は第二形態) first without user manually selecting it. For example, searching `sodom` will give 3 forms of sodom, but being same unit, because all 3 forms contain word `sodom`. True form search mode filters out such overlapped unit, picking 3rd or 2nd form\n\nこの検索モードのを有効にするために`p!fs`に`-tf`を渡す必要がなくなります",
+  "config_truefalse" : "第三形態検索モードではBotが第三形態を自動で選択します(第三形態がない場合は第二形態) For example, searching `sodom` will give 3 forms of sodom, but being same unit, because all 3 forms contain word `sodom`. True form search mode filters out such overlapped unit, picking 3rd or 2nd form\n\nこの検索モードを有効にするには`p!fs`に`-tf`を渡す必要があります",
```

Comment:
I recommend you to translate full sentences so that we won't release WIP texts to users. So I request you to finish up the translation. If you don't know what it means, please contact me in support server